doc/developer/northbound/plugins-sysrepo.rst

```diff
@@ -1,140 +1,192 @@
-Plugins Sysrepo (old)
-=====================
+Plugins Sysrepo
+===============
 
 Installation
 ------------
 
 Required dependencies
 ^^^^^^^^^^^^^^^^^^^^^
+Install FRR build required dependencies, check Building FRR document for specific platform required packages.  
```

Comment:
Can we reference `check Building FRR document`  directly to ease the navigation?

Reply:
done